crates/ruff_linter/src/rules/perflint/snapshots/ruff_linter__rules__perflint__tests__preview__PERF403_PERF403.py.snap

```diff
@@ -351,3 +353,24 @@ PERF403.py:166:13: PERF403 [*] Use `dict.update` instead of a for-loop
 165     |-        if lambda: 0:
 166     |-            dst[k] = v
     164 |+    dst.update({k: v for k, v in src if (lambda: 0)})
+167 165 | 
+168 166 | # https://github.com/astral-sh/ruff/issues/18859
+169 167 | def foo():
+
+PERF403.py:172:9: PERF403 [*] Use a dictionary comprehension instead of a for-loop
```

Comment:
This PR does seem to change the rules that get reported because this wasn't a violation before (when building in release build)

Reply:
Are you talking about the ecosystem changes? If so, It doesn't make sense for `RUF043` to be triggering here, it doesn't make use of code changed in this PR.